src/routes/server.js

```diff
@@ -1,19 +1,9 @@
 const axios = require('axios');
-const coursePermissions = require('./server.coursePermissions');
-const me = require('./server.me');
 
-const server = (app) => {
+module.exports = (app) => {
 	const { server: { baseURL }, timeout } = app.get('routes');
-	const axiosServer = axios.create({
+	return axios.create({
```

Comment:
Das man die config nur über app.get bekommt ist in dem punkt echt doof. Was mir als schnelle lösung einfällt ist, ich mir aber nicht ganz sicher bin ob es functionniert ist: Du gibst bei module.exports keine function zurück sondern ein opject mit der function die app übergeben bekommt und die das axios element erzeugt und als zweiten parameter den axios server (default als ein leeres object). Das leere object wird beim start durch das aufrufen der function ersetzt durch den eigentlichen axios server.

Reply:
können wir gleich noch drüber reden, ansonsten würde ich es vorerst einfach mal so lassen und dann irgendwann später noch mal drauf schauen